storage/kubernetes/client.go

```diff
@@ -97,7 +99,19 @@ func (cli *client) urlFor(apiVersion, namespace, resource, name string) string {
 	if strings.HasSuffix(cli.baseURL, "/") {
 		return cli.baseURL + p
 	}
-	return cli.baseURL + "/" + p
+
```

Comment:
If URL has a suffix /, then limits will not be applied.
https://github.com/dexidp/dex/blob/8ffb53869ec552605fb1af6816bb3936e0963aec/storage/kubernetes/client.go#L99-L101

Reply:
fixed